Added Dockerfile for easy RHEL 7 testing.

pcseanmckay opened the pull request:
Dockerfile added to create basic RHEL 7 test docker image.  Current Dockerfile pulls RHEL 7 image, installed EPEL and IUS repositories, installs git2u, installs vim, and installs bzip2.

Commit:
Added Dockerfile for easy RHEL 7 testing.

ragingpastry reviewed (commented):
I'd suggest collapsing these commands into one layer and then clearing the yum cache. This keeps the image as slim as possible.

For example:
RUN yum update -y && yum install .......... && yum clean all

Commit:
Updated Dockerfile to try and reduce layers and size.

Commit:
Updated Dockerfile.

Updated Dockerfile to install EPEL and IUS on the same command.  Also
added missing packages that are required by bashellite, except for
docker.

Commit:
Fixed Dockerfile to install createrepo.
